docs/02-for-app-authors/02-requirements.md

```diff
@@ -30,6 +33,72 @@ Every repository must contain a manifest describing how to build the application
 
 This manifest may import other manifest files in the same repository. It's quite common to add the [shared-modules](shared-modules) repository as a submodule.
 
+#### Example manifest
+
+This assumes that upstream brings appdata, icons and desktop files and installs them via their post build install. In this case we're using `meson` as the buildsystem.
+
+<Tabs groupId="manifest-language" defaultValue="json" queryString>
+  <TabItem value="json" label="json" default>
+
+```json title="io.github.example.MyCoolApp.json"
+{
+  "id": "io.github.example.MyCoolApp",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "mycoolapp",
+  "finish-args": [
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--share=ipc"
+  ],
+  "modules": [
+    {
+      "name": "mycoolapp",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/example/mycoolapp/releases/download/v1.0.0/mycoolapp-source-1.0.0.tar.xz",
```

Comment:
Let's also put x-checker-data here, so that more people are willing to make use of it to automatically update their packages. More comments in the manifest would also help.

Reply:
Which unfortunately raises the question, which one. I would go for anitya, as it's one of the more simpler ones. But hint at the existance of others

Reply:
anitya is fine by me.